gcp/appengine/main.tf

```diff
@@ -0,0 +1,14 @@
+resource "google_project_service" "project" {
+  project = "${var.project_id}"
+  service = "appengine.googleapis.com"
+
+  disable_dependent_services = true
+}
+
+resource "google_app_engine_application" "app" {
+  project     = "${var.project_id}"
+  location_id = "${var.location_id}"
+  depends_on = [
```

Comment:
Why do we need this? It's not recommended to use it as Terraform should handle the dependencies on it's own.

Reply:
If removed, the creation of this resourse starts at the same time as the api service being enabled, so It fails. Terraform does not see it as a dependency so I had to add it manually